Setup.sh

```diff
@@ -1,144 +1,94 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-# EXPERIMENTAL!
-# This file is experimental and is not maintained directly by Improbable. Please use at your own risk.
-
-set -e -u -o pipefail
+set -e -u -o pipefail -x
```

Comment:
```suggestion
set -e -u -o pipefail
[[ -n "${DEBUG:-}" ]] && set -x
```

Optionally very verbose instead of always so